src/Engine/Component/PathfindingComponents/PathfindingComponent.hpp

```diff
@@ -29,6 +29,9 @@ class PathfindingComponent : public Component {
 
     virtual void update(float) override;
 
+    // TODO : just add enable/disable options for all components?
```

Comment:
yea this would be ideal. I think I know a good way to implement it, so I'll do that